src/main/java/org/spongepowered/common/service/pagination/SpongePaginationBuilder.java

```diff
@@ -28,23 +28,27 @@
 import static com.google.common.base.Preconditions.checkState;
 
 import com.google.common.collect.ImmutableList;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.TextComponent;
+import org.checkerframework.checker.nullness.qual.NonNull;
 import org.spongepowered.api.service.pagination.PaginationList;
-import org.spongepowered.api.text.Text;
 
 import javax.annotation.Nullable;
 
 public class SpongePaginationBuilder implements PaginationList.Builder {
 
+    private static final @NonNull TextComponent EQUALS_TEXT = TextComponent.of("=");
+
     private final SpongePaginationService service;
     @Nullable
-    private Iterable<Text> contents;
+    private Iterable<Component> contents;
     @Nullable
-    private Text title;
+    private Component title;
     @Nullable
-    private Text header;
+    private Component header;
     @Nullable
-    private Text footer;
-    private Text paginationSpacer = Text.of("=");
+    private Component footer;
+    private Component paginationSpacer = EQUALS_TEXT;
```

Comment:
Static acesses from the same class should also be qualified with the class name.